src/Collections-Strings-Tests/StringTest.class.st

```diff
@@ -1654,6 +1654,17 @@ StringTest >> testMatch [
 		self deny: (each match: 'foo') ]
 ]
 
+{ #category : #tests }
+StringTest >> testNormalizeCamelCase [
+
+	self assert: '' normalizeCamelCase equals: String empty.
+	self assert: 'CamelCase' normalizeCamelCase equals: 'Camel Case'.
+	self assert: 'CamelCase123' normalizeCamelCase equals:'Camel Case123'.
+	self assert: 'ABC' normalizeCamelCase equals: 'ABC'.
+	self assert: 'TresTristesTigres' normalizeCamelCase equals: 'Tres Tristes Tigres'.
+	self assert: 'TresTristesTigresABC' normalizeCamelCase equals: 'Tres Tristes Tigres ABC'
```

Comment:
please add at least one test with two wording parts as input **separated by space** to cover that case

self assert: 'CamelCase HelloWorld' normalizeCamelCase equals: ...

Reply:
Added